crates/symbolicator-service/src/caching/memory.rs

```diff
@@ -168,6 +168,10 @@ pub trait CacheItemRequest: 'static + Send + Sync + Clone {
     /// Loads an existing element from the cache.
     fn load(&self, data: ByteView<'static>) -> CacheContents<Self::Item>;
 
+    fn refresh(&self, _contents: &CacheContents<Self::Item>) -> Option<Self::Item> {
```

Comment:
Can you add docs to this, what is it used for, when is it called?

Reply:
This is something I was toying with that I committed by mistake.